### ISSUE TITLE
the "streams" exercise needs more details and hints

### ISSUE THREAD

**Issue by Pomax** [hapijs/makemehapi#126]
people new to hapi are likely not to be Node experts, and thus entirely likely to never have worked with streams and the `.pipe()` function before.

While the hints do cover how to create a stream, it does not cover how the rot13-transform library is supposed to be used in conjunction with streams, nor is a link provided for reading the API for this library, making "doing the assignment" more of a "hunt for other tutorials to explain how to use streams and the indicated library".

It would be a good idea to at least explain how to use streams with `.pipe()` and what not to do when you do read up on streams (e.g. don't end with a `pipe(process.stdout)`, which you might be tempted to do when you read https://www.npmjs.com/package/rot13-transform in order to find out how to use it, because Hapi will be adding more pipes)

**Comment by nvcexploder** [comment]
Do you think that this merits being split into a new workshopper? I have been out on Holiday, but some things that @nelsonic had mentioned before have really resonated. I feel there needs to be at least two: one specifically for beginners that have less node and/or web experience, and one for folks that are prepared to go a bit deeper into the framework.

**Comment by Pomax** [comment]
Not really. Just work people up through the exercises. If you know streams, then this should be an easy assignment, if you don't, the assignment text and hints can probably be phrased so that even someone unfamiliar with streams at least "gets" the idea behind them while doing the assignment.
